config/routes/annotations.yaml

```diff
@@ -1,7 +1,7 @@
 controllers:
```

Comment:
As you stated, should we not rename the `annotations.yaml` filename to `controllers.yaml`?